examples/docker-for-mac/my_test.sh

```diff
@@ -66,9 +66,9 @@ configure_kubectl() {
     port=$(get_apiserver_arg "$apiserver_id" --secure-port)
 
     docker cp "$HOME/.kube" "$testcontainer_id:/root/.kube"
-    docker exec "$testcontainer_id" kubectl config set-cluster docker-for-desktop-cluster "--server=https://$ip:$port"
-    docker exec "$testcontainer_id" kubectl config set-cluster docker-for-desktop-cluster --insecure-skip-tls-verify=true
-    docker exec "$testcontainer_id" kubectl config use-context docker-for-desktop
+    docker exec "$testcontainer_id" kubectl config set-cluster docker-desktop "--server=https://$ip:$port"
```

Comment:
I'm using the `2.0.2.1` version of docker desktop for mac on the "edge" channel, and my context is named `docker-for-desktop`. Is there a release note about the change?

Reply:
same here `2.0.2.1 (31274)`.
deleted `config`, recreated docker4mac k8s cluster, I still see `docker-desktop`

Reply:
``` 
.kube $ cat config
apiVersion: v1
clusters:
- cluster:
    insecure-skip-tls-verify: true
    server: https://localhost:6443
  name: docker-desktop
contexts:
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-desktop
- context:
    cluster: docker-desktop
    user: docker-desktop
  name: docker-for-desktop
current-context: docker-desktop
kind: Config
preferences: {}
users:
- name: docker-desktop
  user:
...
```

Reply:
right, there are two contexts pointing to the same cluster, context can be `docker-for-desktop` or `docker-desktop`, but `kubectl config set-cluster ...` commands must use `docker-desktop` otherwise it fails

Reply:
My bad, I just realized I haven't had the docker desktop cluster enabled for a long time and I was just seeing a stale context. Enabling kubernetes support in the UI gave me the same config.

Reply:
yup, it happened for me too, haven't used it for a while as well